Add `Perak osu! Tournament 2022` wiki article (#7239)
ppy/osu-wiki: wiki/Tournaments/POT/2022/en.md

---
tags:
  - POT
  - POT 2022
---

# PERAK osu! Tournament 2022

![PERAK osu! Tournament 2022 logo](img/banner.jpg)

> The **PERAK osu! Tournament 2022** was a double-elimination 1v1 osu! tournament hosted by ![][flag_ID] [Splacten](https://osu.ppy.sh/users/10018405) as part of the annual [PERAK Games League](https://perak.cs.ui.ac.id/) festival. The tournament was open to all players who were registered as a student at (or as an alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) regardless of rank. It was the second iteration of the PERAK osu! Tournament.

Reviewer: ```suggestion
The **PERAK osu! Tournament 2022** (***POT 2022***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_ID] [Splacten](https://osu.ppy.sh/users/10018405) as part of the annual [PERAK Games League](https://perak.cs.ui.ac.id/) festival. The tournament was open to all players who were registered as a student at (or as an alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) regardless of rank. It was the second iteration of the PERAK osu! Tournament.
```

> The **PERAK osu! Tournament 2022** (***POT 2022***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_ID] [Splacten](https://osu.ppy.sh/users/10018405) as part of the annual [PERAK Games League](https://perak.cs.ui.ac.id/) festival. The tournament was open to all players who were registered as a student at (or as an alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) regardless of rank. It was the second iteration of the PERAK osu! Tournament.

Reviewer: ```suggestion
The **PERAK osu! Tournament 2022** (***POT 2022***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_ID] [Splacten](https://osu.ppy.sh/users/10018405) as part of the annual [PERAK Games League](https://perak.cs.ui.ac.id/) festival. The tournament was open to all players who were registered as a student at (or alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) regardless of rank. It was the second iteration of the PERAK osu! Tournament.
```

| Commentator | ![][flag_ID] [Victim\_Crasher](https://osu.ppy.sh/users/2084869), ![][flag_ID] [Tix](https://osu.ppy.sh/users/11421465), ![][flag_ID] [nasje](https://osu.ppy.sh/users/7579498), ![][flag_ID] [eucharistica](https://osu.ppy.sh/users/28736472) |
| Referee | ![][flag_ID] [kilgo](https://osu.ppy.sh/users/9692053), ![][flag_ID] [Romizzz](https://osu.ppy.sh/users/6154769) |
| Graphic designer | ![][flag_ID] [Kizito Hiro](https://osu.ppy.sh/users/2263272) |
| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |

## Links

- **[Master spreadsheet](https://docs.google.com/spreadsheets/d/175mrODHUOZIUhW6S3s86T7sZnGfy35-KHht8v2c0ixI/edit#gid=0)**
- [Official website](https://perak.cs.ui.ac.id/leaderboard/13)
- [Discord server](https://discordapp.com/invite/8vGTedxKmD)
- [Livestream channel](https://www.twitch.tv/perakgamesleague)
- [Challonge brackets](https://challonge.com/PERAKOSU2022)

## Participants

Listed below are the players who were participating in the tournament along with their respective seed number and average percentage across all ten Qualifier beatmaps.

> | Seed No. | Player | Avg. Percentage |
> | :-- | :-- | :-- |

Reviewer: ```suggestion
| Seed | Player | Avg. percentage |
| :-: | :-- | :-- |
```

### Round of 12

Sunday, 6 March 2022:

| Bracket | Player 1 |  |  | Player 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Upper | **[Stingray](https://osu.ppy.sh/users/2215822)** ![][flag_ID] | **5** | 0 | ![][flag_ID] [larrypult](https://osu.ppy.sh/users/12255948) | [#1](https://osu.ppy.sh/community/matches/98469707) |
| Upper | **[NeoZap](https://osu.ppy.sh/users/10024869)** ![][flag_ID] | **5** | 0 | ![][flag_ID] [DeltaE](https://osu.ppy.sh/users/8361921) | *win by default* |

Monday, 7 March 2022:

| Bracket | Player 1 |  |  | Player 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Upper | [hamdanpapeng](https://osu.ppy.sh/users/10854143) ![][flag_ID] | 1 | **5** | ![][flag_ID] **[nasje](https://osu.ppy.sh/users/7579498)** | [#1](https://osu.ppy.sh/community/matches/98501050) |

## Ruleset

### General rules

1. Beatmap scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
2. The mappools for each round will be announced by the Tournament Management in advance before the actual matches take place.

> 3. Match schedules will be predetermined by the Tournament Management. If there are any player(s) who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the `#scheduling` channel in the tournament's Discord server.

Reviewer: ```suggestion
3. Match schedules will be predetermined by the Tournament Management. If there are any players who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the `#scheduling` channel in the tournament's Discord server.
```

4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
5. If a player does not show up within **10 minutes** of the start time, their opponent wins by default.
6. If no staff or referee is available, the match will be postponed.
7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards players who perform better in general during the course of the beatmap regardless of their remaining health at the end.
8. If a player disconnects, it will be treated as if they had failed the beatmap. 
   - A match can be rematched for disconnects that occur within a few seconds after the beatmap has been started by the referee.
9. If a player disconnects between beatmaps, the match can be delayed up to 15 minutes at most. 

>    - In case the disconnected player fails to report back after the maximum allowance of 15 minutes had passed, the opposing player may be declared to win the match by default.

Reviewer: ```suggestion
   - In case the disconnected player fails to report back after the maximum allowance of 15 minutes pass, the opposing player may be declared to win the match by default.
```

10. Lag is not a valid reason to nullify a beatmap.
11. If any problems during the match occur, the Tournament Management will make a decision based on the referee's report.
12. It is expected that all players be polite and respectful to each other. Penalties will be given upon violation.

>     - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from the future iterations of the tournament by the Tournament Management.

Reviewer: ```suggestion
    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from future iterations of the tournament by the Tournament Management.
```

>     - Usage of any tools and programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.

Reviewer: ```suggestion
    - Usage of any tools or programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
```

### Tournament registration

1. All interested players are required to register into the tournament individually.

>    - In order to be eligible to play in the tournament, a player must be able to attest their status as a registered student at (or as an alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) by showing a valid [EMAS UI credential](http://emas.ui.ac.id) to the Tournament Management.

Reviewer: ```suggestion
   - In order to be eligible to play in the tournament, a player must be able to attest their status as a registered student at (or alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) by showing a valid [EMAS UI credential](http://emas.ui.ac.id) to the Tournament Management.
```

Author: hmm i'll leave this one untouched to keep things consistent with what's written in the opening paragraph (Line 11) i think...

Reviewer: well i suggested the same thing on the 2021 article and you applied that one sooo....

Author: oh D:

okok will apply this

2. To ensure valid and serious registrations, every registered player will be checked by the Tournament Management.

> 3. The list of players who are deemed to be eligible to compete in the tournament will be published by the Tournament Management after the Registration Phase has ended.

Reviewer: ```suggestion
3. The list of players who are deemed to be eligible to compete in the tournament will be published by the Tournament Management after the Registration phase has ended.
```

> 4. Testplayers, referees, and map selectors may not participate as players in this tournament.

Reviewer: ```suggestion
4. Test players, referees, and mappool selectors may not participate as players in this tournament.
```

### Round-specific rules

#### Qualifier rules

1. Each player will have to sign up to one of the twenty Qualifier lobbies that have been scheduled and prepared by the Tournament Management in advance.
2. In the lobby, players will have to consecutively play all of the ten Qualifier beatmaps in the order of NM1 → NM2 → NM3 → NM4 → HD1 → HD2 → HR1 → HR2 → DT1 → DT2.
3. Players **are not allowed** to ban any beatmaps in the Qualifiers.
4. Players **are not allowed** to join in (or register for) more than one Qualifier lobbies.

> 5. Based on their performance in the Qualifier, players will be placed in seeds based on their average percentage across all ten qualifier beatmaps following [this formula](http://www.indeed.com/career-advice/career-development/how-to-calculate-average-percentage).

Reviewer: ```suggestion
5. Based on their performance in the Qualifier, players will be placed in seeds based on their average percentage across all ten Qualifier beatmaps following [this formula](http://www.indeed.com/career-advice/career-development/how-to-calculate-average-percentage).
```

6. Failure to attend in any of the twenty Qualifier lobbies will result in an instant elimination from the tournament.

#### Knock-out stage rules

1. All will be matched to each other based on their Qualifiers seeding.
2. Players will compete against each other using the Double Elimination system.

> 3. The Double Elimination System works as follows:

Reviewer: ```suggestion
3. The double-elimination system works as follows:
```

>    - Players who lose in the Upper Bracket can still play again on the Lower Bracket.
>    - Players who lose in the Lower Bracket will be eliminated from the tournament.

Reviewer: (really need to open an issue to standardise tournament terms soon)

```suggestion
   - Players who lose in the upper bracket can still play again on the lower bracket.
   - Players who lose in the lower bracket will be eliminated from the tournament.
```

>    - In the Grand Final, the winner of the the Upper Bracket will only need to win a single match against their opponent in order to claim the championship title. The winner of the Lower Bracket, however, will need to win two matches and enforce a Bracket Reset against their opponent in order to claim the championship title.

Reviewer: ```suggestion
   - In the Grand Final, the winner of the upper bracket will only need to win a single match against their opponent in order to claim the championship title. The winner of the lower bracket, however, will need to win two matches and enforce a bracket reset against their opponent in order to claim the championship title.
```
